### PR TITLE
Solved issue #522

### PR DESCRIPTION
**Pull request by aliya-rahmani**
Simplified language in translation page

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@aliya-rahmani

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: documenting.rst [hunk 1]
```diff
@@ -1790,10 +1790,9 @@ Ask on doc-sig, or better, make a PR on the `devguide
 I have a translation, but not on git, what should I do?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Just ask for help on the doc-sig mailing list and our python-fu, git-fu
-and bash-fu combined will help you create an appropriate repository. If
-you use a tool like transifex don’t worry.  Keeping them in sync is not
-that hard.
+Ask for help on the doc-sig mailing list and our python console, git
```
Comment (willingc) [inline documenting.rst:1793]:
```suggestion
Ask for help on the doc-sig mailing list
```

File: documenting.rst [hunk 2]
```diff
@@ -1790,10 +1790,9 @@ Ask on doc-sig, or better, make a PR on the `devguide
...
+and bash command line interface combined will help you create an appropriate repository.
```
Comment (willingc) [inline documenting.rst:1794]:
```suggestion
about creating an appropriate repository. 
```

File: documenting.rst [hunk 3]
```diff
@@ -1790,10 +1790,9 @@ Ask on doc-sig, or better, make a PR on the `devguide
 I have a translation, but not on git, what should I do?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Just ask for help on the doc-sig mailing list and our python-fu, git-fu
-and bash-fu combined will help you create an appropriate repository. If
-you use a tool like transifex don’t worry.  Keeping them in sync is not
-that hard.
+Ask for help on the doc-sig mailing list and our python console, git
+and bash command line interface combined will help you create an appropriate repository. 
+Using a tool like transifex can help keeping them in sync.
```
Comment (willingc) [inline documenting.rst:1795]:
```suggestion
Using a tool like transifex can help keep translations in sync.
```

**Review by willingc — COMMENTED** [review]
Hi @aliya-rahmani. Thanks for the documentation change. Please submit the CLA if you haven't already done so. It will automatically update to signed status a day or two after it is submitted.

**Comment by aliya-rahmani** [comment]
@willingc I've done the recommended changes and signed CLA as well.

**Comment by hugovk** [comment]
Looks like the 2 commits from https://github.com/python/devguide/pull/628 have been accidentally included here.

Please could you undo those two from here?

**Comment by aliya-rahmani** [comment]
@hugovk I've undone the changes and now the pull request is having 7 commits.

**Comment by willingc** [comment]
Hi @aliya-rahmani, I've merged in your PR #630 and left you a link to a section of the devguide. Would you be able to update this PR to only contain those that relate to `documenting.rst`?

Remember, for the future, when creating a new change it's important to develop on a new branch instead of `master`. This will minimize merge conflicts and extraneous files in your new branch when you open a PR. 👍

**Comment by aliya-rahmani** [comment]
Hi @willingc, thank's alot for the guidelines, I'm a begineer so just having some issues with understanding things. I've made a new PR #638 for the issue, plz have a look.

**Pull request closed without merging**